boards/arm/mimxrt1020_evk/board.cmake

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2018, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+board_runner_args(jlink "--device=MCIMXRT1021")
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
```

Comment:
pyOCD also now support the mimxrt1020 target, it's therefore possible to add support for it similar to what has been done for mimxrt1050_evk.

Reply:
Note that pyOCD has an issue programming flash on imx rt boards. I've been working on a fix, but it's not stable enough to upstream yet. I got program to work, but broke erase in the process. :(

Reply:
I only tried it for executing from RAM. There is no support for XIP yet on this board, so I haven't tried that.